packages/cluster-ui/src/sessions/sessionsPage.tsx

```diff
@@ -102,11 +103,13 @@ export class SessionsPage extends React.Component<
     const searchParams = new URLSearchParams(history.location.search);
     const sortKey = searchParams.get("sortKey") || undefined;
```

Comment:
Is there still a point of keeping `sortKey` now that we are sorting by `columnTitle` ?

Reply:
Removed